apps/verticalhome/js/app.js

```diff
@@ -219,6 +221,12 @@
           window.removeEventListener('gaiagrid-layout-ready', this);
           break;
 
+        case 'visibilitychange':
+          setTimeout(function() {
+            document.body.style.overflow = document.hidden ? 'hidden' : '';
```

Comment:
p.s. this was something I meant to have a look at to do during rotation transitions too - we should totally do it there also...